src/frontend/src/routes/(app)/+layout.svelte

```diff
@@ -24,10 +25,12 @@
 	back={route === 'settings' ? 'header' : route === 'transactions' ? 'hero' : undefined}
 />
 
-<main class="pb-5 pt-8 sm:pb-12">
-	<LoadersGuard>
-		<slot />
-	</LoadersGuard>
-</main>
+<AuthGuard>
```

Comment:
We want the same login experience on both "Tokens" page as on any other pages ("Settings" and "Transactions"). Correct?

Reply:
Correct, on all pages. It becomes really a "Landing Page", less of "App Login"